cmd/crc/cmd/json.go

```diff
@@ -19,17 +19,17 @@ func addOutputFormatFlag(cmd *cobra.Command) {
 }
 
 type prettyPrintable interface {
-	prettyPrintTo(writer io.Writer) error
+	prettyPrintTo(writer io.Writer, err error) error
```

Comment:
This extra error feels weird. I understand it's not really convenient to pass the error. 

I made an attempt with the use of Unwrap() method: https://github.com/code-ready/crc/compare/master...guillaumerose:unwrap?expand=1#diff-b6063933a261de0116a269a05888f288ccde569b5e83c2abe9435f79a65fc46bR65

Reply:
@guillaumerose I will look into the suggestion tomorrow but if you see when you are using render it is having the obj which contains the error as string so passing same err as different argument shouldn't be the issue.

Reply:
I did not look in detail at Guillaume's code, but upon seeing 
```
	if s.Error != "" {
		return errors.New(s.Error)
	}
```
I wanted to ask why `s.Error` is not a real error, which is what Guillaume's changes are doing, so imo it's worth taking a closer look ;)